security/cmd/istio_ca/main.go

```diff
@@ -144,6 +148,9 @@ func init() {
 	flags.DurationVar(&opts.LivenessProbeOptions.UpdateInterval, "livenessProbeInterval", 0,
 		"Interval of updating file for the liveness probe.")
 
+	flags.DurationVar(&opts.probeCheckInterval, "probeCheckInterval", defaultProbeCheckInterval,
+		"Interval of checking the liveness of the service.")
```

Comment:
What about Interval of checking the liveness of the *CA*, which is more accurate.

Reply:
Done